test/test_mite_websockets.py

```diff
@@ -29,28 +29,25 @@ async def dummy_journey(ctx):
         await ctx.websocket.connect("wss://foo.bar")
 
     with patch("websockets.connect", new=connect_mock):
-        await dummy_journey(context)
+        wb = await dummy_journey(context)
```

Comment:
Why are we capturing the return value of the function?  (Which will always be `None`)